Makefile

```diff
@@ -1,6 +1,7 @@
 IMAGE_NAME ?= owncloudci/core:latest
```

Comment:
Just drop the Makefile? This is anyway not used from anybody manually?

Reply:
Don't know. Some repos had it some not. I decided to keep/add it for all